### PR TITLE
Fixes for the development site

### PR DESCRIPTION
**Pull request by mattwatsoncodes**
Fixes a couple of issues occurring on the dev site in order to fix maps and styles.

- Security exceptions are needed for blob:https://wikimediafoundation-org-develop.go-vip.co/ so that the map works correctly
- Use of mark for selecting inline font colours has a yellow background, needs to be made transparent
- Adds a cursor on hover for expandable buttons
- Fixes image spacing in one of the patterns

### CODE REVIEW

File: inc/assets.php [hunk 1]
```diff
@@ -48,7 +49,7 @@ function( $carry, $port ) {
 
 	$headers['Content-Security-Policy'] = preg_replace(
 		"/script-src 'self'/",
-		"script-src 'self' blob: https://wikimedia.vipdev.lndo.site",
+		"script-src 'self' blob: https://wikimedia.vipdev.lndo.site https://wikimediafoundation-org-develop.go-vip.co",
```
Comment (kadamwhite) [inline inc/assets.php:52]: Can we add this to the security integration plugin in the Foundation site project instead? It is not specific to this plugin, technically

File: src/blocks/expandable/frontend.scss [hunk 2]
```diff
@@ -23,6 +23,7 @@
 .expandable-expander {
 	background-color: #fff;
 	border: 1px solid;
+	cursor: pointer;
```
Comment (kadamwhite) [inline src/blocks/expandable/frontend.scss:26]: Broadly I understand the argument here, but to borrow some resources from an old debate about this in the Tailwind GH report: while it's common to give `<button>`s `cursor: pointer;`, this is an anti-pattern. The [CSS spec](https://drafts.csswg.org/css-ui-3/#valdef-cursor-pointer) establishes that `cursor: pointer` indicates an element is a link.

This article https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b has some additional justification for this.

**Review by kadamwhite — APPROVED** [review]
Approving for convenience but left some notes on specific changes

File: src/blocks/expandable/frontend.scss (hunk 2 above)
Comment (mattwatsoncodes) [reply src/blocks/expandable/frontend.scss:26]: TIL 🤯 thank you @kadamwhite 🙏

File: inc/assets.php (hunk 1 above)
Comment (mattwatsoncodes) [reply inc/assets.php:52]: Sure, I'll add a ticket to add that in and remove it from here 👍, I'll leave it in now so that we can demo.

**Pull request merged**